.travis.yml

```diff
@@ -10,4 +10,4 @@ install:
   - travis_retry composer install --no-interaction
 
 script:
-  - vendor/bin/phpunit
+  - vendor/bin/phpunit -c ./phpunit.xml --coverage-text
```

Comment:
You added this to a previous PR as well. Same comment here. Do we want this? Scrutenizer is currently setup to calculate code coverage.

Reply:
That wat trickled down from the previous PR. I feel adding code coverage adds some runtime, and if Scrutenizer is going to cover this too. I'm ok with removing it here.

Reply:
OK, Let's remove coverage from travis then.